assets/css/components/icons.css

```diff
@@ -0,0 +1,3 @@
+/* Bootstrap glyphicons */
```

Comment:
FYI: if you rename the `icons.scss` to `_icons.scss`, then this file is not created anymore.